graphics/libfreehand/Portfile

```diff
@@ -30,8 +34,12 @@ depends_lib         port:librevenge \
                     port:lcms2 \
                     port:zlib
 
+# blacklist compilers with known problems on older systems
+compiler.blacklist-append  *gcc-3.* *gcc-4.* {clang < 300}
+
 configure.args      --disable-werror \
                     --disable-silent-rules \
+                    --disable-tests \
```

Comment:
I'm curious why we want to disable tests? (We've been making an effort to make tests available more of the time.)

Reply:
cppunit, needed for the tests, is a bit of a PITA. Also, IMHO, for port releases, I think tests should be disabled / not built. I think the tests are mostly for the maintainer / committer to run and be sure what's happening, but end-users shouldn't need to worry about them in the release by default. I have offered them as a variant.

Reply:
As per the title, these are what I propose as minimal build fixes.  There's no point in enabling tests if the Port doesn't use them.  To enable the tests a dependency on cppunit needs to be added and the test phase setup properly.  My intent was to leave this as exercise for the maintainer and is a bit more than I am comfortable in doing openmaintainer.  Otherwise, I would have just done the full thing and committed the changes.  I would prefer to see the maintainer do it.

Reply:
It built on 10.6.8 for me with an unmodified Portfile, with clang-4.0 against libc++. So the PG should fix it (unless there are weird math errors). I'd say commit it, and I'll fix what is fixable if any errors show up.

Reply:
I agree.  If this works then the same can be done for libmspub and libpagemaker.